app/controllers/miq_policy_rsop_controller.rb

```diff
@@ -80,18 +80,18 @@ def rsop
   def rsop_option_changed
     assert_privileges('policy_simulation')
     if params[:event_typ]
```

Comment:
I think this code is wrong, because `if params[:event_typ]` implies presence, so the logic for resetting forms is currently lost with these changes.  In the case that the user explicitly chose the "Choose", we need to handle that case by clearing the other dropdowns and values.

@MelsHyrule Do you know if when the user selects "Choose", does `params[:event_type]` comes in as nil or just not at all?  If it comes in as nil, I think we need to do:

```suggestion
    if params.key?(:event_typ)
```

Reply:
Specifically for `params[:event_type]` this comes in as `""` when you select `"Choose"` and it comes in as `nil` every-time you select anything in any of the other 3 drop-downs. It only has a numeric value when you select from the `Event Selection - Type dropdown` something that isn't `"Choose"`. The same logic can be applied to the other drop-downs.